ckanext/switzerland/logic.py

```diff
@@ -292,6 +298,45 @@ def ogdch_showcase_search(context, data_dict):
         raise NotFound
 
 
+@side_effect_free
+def ogdch_harvest_monitor(context, data_dict):
+    """Returns the status of the fetch and gather processes.
+
+    If there are still-running harvest jobs that were created more than
+    6 hours ago, it is likely that either of those processes has stopped
```

Comment:
hmm, it kind of urked me a little bit to see 6 hard-coded like that. But for a first endpoint that is probably sufficient.
I thought if it was configured in the ini-file that would be cool, but it might be most useful to be able to run it with a parameter (e.g. `ogdch_harvest_monitor?hours=6`) but probably out of scope, is it? If you think this is something super simple, that would be cool, but I think we can also leave it for now :)